src/Smalot/PdfParser/PDFObject.php

```diff
@@ -292,7 +292,7 @@ public function getText(Page $page = null)
                             )
                         ) {
                             // horizontal offset
-                            $text .= ' ';
+                            //$text .= ' ';
```

Comment:
What did you changed here? It looks the same. Furthermore please remove commented-out code.